MachOKit/Segments/MKMachO+Segments.m

```diff
@@ -109,6 +109,7 @@ - (NSDictionary*)_segments
             }
         }
         
+        [MKBackedNode sortOptionalNodeArray:(NSMutableArray *)segments];
```

Comment:
I prefer to keep the 'All' segments array matching the order of the load commands.  

Create a copy of the segments array, sort it, and add it to the _segments dictionary under a new key (e.g, MKSortedSegments).  This will eliminate the need to handle sorting optionals - any optional without a value can be discarded from the sorted array.

Reply:
Done in bcac3dc